gstd/src/msg/async.rs

```diff
@@ -159,3 +159,99 @@ pub fn send_bytes_and_wait_for_reply<T: AsRef<[u8]>>(
 
     Ok(MessageFuture { waiting_reply_to })
 }
+
+/// Create a program and wait for reply.
+///
+/// This function works similarly to
+/// [`create_program_wbytes_and_wait_for_reply`], with one difference - it takes
+/// the structure in, then encodes it and sends it in bytes. The program will be
+/// interrupted (waiting for a reply) if an `.await` has been called on the
+/// `CodecMessageFuture` object returned by the function.
+#[deprecated(note = "please use `gstd::proc::create_program` instead")]
+pub fn create_program_and_wait_for_reply<E: Encode, D: Decode>(
```

Comment:
```suggestion
pub fn create_program_for_reply<E: Encode, D: Decode>(
```
We now follow naming wouth `and_wait`